### PR TITLE
Loa 341 implement badge page styling

### PR DESCRIPTION
**Pull request by Kostanix**

### CODE REVIEW

**Review by JohannesHausch — CHANGES_REQUESTED** [review]
The green of the progressbar hurts in my eyes. Can you please change that?

File: app/src/main/res/layout/fragment_badges_page.xml [hunk 1]
```diff
@@ -95,60 +100,29 @@
...
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            <ImageView
+                android:id="@+id/badge_page_map_button"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="@drawable/map"
+                android:contentDescription="@string/image_frame"
+                android:scaleType="center"
```
Comment (ayal104) [inline app/src/main/res/layout/fragment_badges_page.xml:119]: Why is the content description image_frame?

**Review by ayal104 — CHANGES_REQUESTED** [review]

**Comment by JohannesHausch** [comment]
The space between the icons and the bath name is too small.

File: app/src/main/res/layout/fragment_badges_page.xml (hunk 1 above)
Comment (Kostanix) [reply app/src/main/res/layout/fragment_badges_page.xml:119]: Because it is just a image frame, vut I changed the name now

**Review by ayal104 — APPROVED** [review]
LGTM!

**Review by JohannesHausch — APPROVED** [review]
LGTM!

**Pull request merged**